SubmittedAssignments/Sanjeevanee_balure_03/index.html

```
@@ -0,0 +1,75 @@
+
+     <div class="container">
+	 
+    <div class="row">
+      <div class="col-7 overflow-auto">
+
+        <invoice-component>
+		
+        </invoice-component>
+      </div>
+      <div class="col-5">
+        <invoice-form-component data='{
```

HTMLHint (1.5.0)

[error] The value of attribute [ data ] must be in double quotes.

(attr-value-double-quotes)